adapter/pkg/adapter/adapter.go

```diff
@@ -18,26 +18,20 @@
 package adapter
 
 import (
-	"time"
-
 	logger "github.com/wso2/product-microgateway/adapter/pkg/loggers"
 	"github.com/wso2/product-microgateway/adapter/pkg/synchronizer"
 )
 
 // GetAPIs function calls the FetchAPIs() with relevant environment labels defined in the config.
-func GetAPIs(c chan synchronizer.SyncAPIResponse, id *string, serviceURL string, userName string, password string,
-	envs []string, skipSSL bool, truststoreLocation string, endpoint string, sendType bool, apiUUIDList []string,
-	requestTimeOut time.Duration, queryParamMap map[string]string) {
+func GetAPIs(c chan synchronizer.SyncAPIResponse, id *string, envs []string, endpoint string, sendType bool, apiUUIDList []string,
+	queryParamMap map[string]string) {
 	if len(envs) > 0 {
 		// If the envrionment labels are present, call the controle plane with labels.
 		logger.LoggerAdapter.Debugf("Environments label present: %v", envs)
-		go synchronizer.FetchAPIs(id, envs, c, serviceURL, userName, password, skipSSL, truststoreLocation,
-			endpoint, sendType, apiUUIDList, requestTimeOut, queryParamMap)
+		go synchronizer.FetchAPIs(id, envs, c, endpoint, sendType, apiUUIDList, queryParamMap)
 	} else {
 		// If the environments are not give, fetch the APIs from default envrionment
-		logger.LoggerAdapter.Debug("Environments label  NOT present. Hence adding \"default\"")
-		envs = append(envs, "default")
```

Comment:
populate Default label again because it is a mandatory field for the API Manager Internal rest APIs.